player/external_files.c

```diff
@@ -110,36 +110,52 @@ static int compare_sub_priority(const void *a, const void *b)
     return strcoll(s1->fname, s2->fname);
 }
 
-static struct bstr guess_lang_from_filename(struct bstr name, int *fn_start)
+static struct bstr guess_lang_from_filename(struct bstr name, int *lang_start)
```

Comment:
I would prefer if we moved this function to language.c and added a test for it. To document it's behavior and avoid breakage in future.

Reply:
That's best done after https://github.com/mpv-player/mpv/pull/13917 to avoid conflicts though I don't think it's bad to have it here since it's specific to external files rather than comparing languages, and there are already other 2 public functions.

Reply:
merged now

Reply:
Done.